src/code/reducers/helpers/load-state-from-authoring.js

```diff
@@ -105,10 +105,24 @@ export function loadStateFromAuthoring(state, authoring, progress={}) {
         template = templateName && templates[templateName];
   if (!template) return state;
 
+  let userChangeableGenes = authoredChallenge.userChangeableGenes;
```

Comment:
That seems ok. When I was initially thinking about `userChangeableGenes` (or even `visibleGenes`, which isn't covered here) having optional trials and named drakes, I was thinking that the names would be up to the templates, instead of hardcoding like it is here. For instance, if one of the intro challenges (one drake visible) wanted to have different genes editable on different trials, it would have to decide that the one drake was called "mother" now.

I think the way to do that would be to update the `split` function to recursively search down the tree and just convert any strings it finds.

That said, in the spirit of YAGNI (you ain't gonna need it), this is all we need now and in all likelihood all we're going to need, so I think this is fine.